src/libcollections/btree/node.rs

```diff
@@ -1359,6 +1359,9 @@ struct MoveTraversalImpl<K, V> {
     is_leaf: bool
 }
 
+unsafe impl<K: Send, V: Send> Send for MoveTraversalImpl<K, V> { }
+unsafe impl<K: Sync, V: Sync> Sync for MoveTraversalImpl<K, V> { }
```

Comment:
But as @Gankro pointed out, `RawItems` doesn't imply ownership so it doesn't implement `Sync` nor `Send`. OTOH `Sync` and `Send` semantics do apply to `MoveTraversalImpl`. figure it should implement both.